index.js

```diff
@@ -1,12 +1,14 @@
 /* eslint-env node */
 'use strict';
 
+const transform = require('./addon/transforms/input');
```

Comment:
Seems vaguely odd to require from `addon` while in node-land. Totally may work fine though. It just surprised me 😝